src/benchmarklib/tpcc/tpcc_random_generator.hpp

```diff
@@ -39,10 +41,10 @@ class TPCCRandomGenerator : public RandomGenerator {
   }
 
   /**
-   * Generates a random last name based on a set of syllables
-   * @param i   a number, if less than 1000, each digit represents a syllable
-   *            for i's greater than 1000 we calculate a non-uniform random number below 1000
-   * @return    a String representing the last name
+   * Generates a random last name based on a set of syllables.
+   * @param i   a number, if less than 1000, each digit represents a syllable for i's greater than 1000 we calculate a
```

Comment:
`i's` not really correct ... _inputs_? or al least just `for an i greater ...`

Reply:
```
   * @param i   given input i, each digit of i is represented as a syllable to form a last name; if input i is larger
   *            than 999, we set i to a random number between 255 and 1000 first
```
?